neo3/wallet/account.py

```diff
@@ -0,0 +1,163 @@
+from __future__ import annotations
+
+import base58
+from typing import Optional
+
+from neo3 import settings
+from neo3.core import types
+
+
+class Account:
+
+    def __init__(self, password: Optional[str] = None,
+                 private_key: Optional[bytes] = None,
+                 nep2_key: Optional[str] = None,
+                 watch_only: bool = False,
+                 script_hash: Optional[types.UInt160] = None,
+                 address: Optional[str] = None
+                 ):
+        """
+        Instantiate an account, this constructor should not be called by the user.
+        """
+
+        self._key = None
+        from neo3.core.cryptography import ECPoint
+        public_key: Optional[ECPoint] = None
+        encrypted_key: Optional[bytes] = None
+
+        if watch_only:
+            if script_hash is not None and address is None:
+                address = self.script_hash_to_address(script_hash)
+            elif address is not None and script_hash is None:
+                address = address
+
+        elif password is not None:
+            from neo3.wallet import private_key_to_nep2
```

Comment:
All imports should be at the top of the file. Only in very exceptional cases (if we really can't break a circular dependency otherwise) do we use inline imports